### ISSUE TITLE
Code and dataset of extremities detection model

### ISSUE THREAD

**Issue by Wang-Yu-Qing** [SoccerNet/sn-calibration#33]
Hi, I want to know if the code of training the extremities detection model is availabel? I only find the [model weights](https://drive.google.com/file/d/1dbN7LdMV03BR1Eda8n7iKNIyYp9r07sM/view?usp=sharing) but I want to checkout the model code and dataset for training these weights. Thanks!

**Comment by fmagera** [comment]
Hello @Wang-Yu-Qing, 
The model is defined [here](https://github.com/SoccerNet/sn-calibration/blob/a57a97deda84e99cde9a5fcd94d6e00dc7d8cc7b/src/detect_extremities.py#L201). The dataset is soccernet-calibration and the dataloader is provided in the dataloader.py 

The training code is not provided but it is a very basic one with a cross entropy loss ! Participants have sometime provided code to train their version of the model, for instance in Jonas Theiner's [work](https://github.com/jtheiner/sn-calibration-segmentation/blob/ffdb3088ffb6c89f28249f330d2e9ca6be3c8094/src/segmentation/train.py). 
Hope this will help !
